package.json

```diff
@@ -30,18 +30,19 @@
     "ember-compatibility-test": "ember try:each"
   },
   "dependencies": {
-    "@embroider/util": "^1.6.0",
+    "@embroider/macros": "^1.6.0",
+    "@embroider/util": "^1.0.0",
     "@glimmer/component": "^1.0.4",
     "broccoli-funnel": "^3.0.3",
     "broccoli-merge-trees": "^4.2.0",
     "broccoli-static-site-json": "^4.4.1",
-    "ember-auto-import": "^1.11.3 || ^2.0.0",
-    "ember-cli-babel": "^7.26.6",
-    "ember-cli-htmlbars": "^5.7.1",
+    "ember-auto-import": "github:mansona/ember-auto-import#non-workspace",
```

Comment:
I assume landing this on a published version will be considered a blocker for rolling out this PR?

Reply:
Yes absolutely! When talking to Ed about https://github.com/ef4/ember-auto-import/pull/530 one of the things I was to check was to see if it "solves my problem" 

I'll also probably end up splitting this PR into a few smaller more focused ones 👍